dll/win32/shell32/wine/PolicyData.h

```diff
@@ -0,0 +1,238 @@
+DEFINE_POLICY( REST_NOTRAYITEMSDISPLAY,          "Explorer",       "NoTrayItemsDisplay"               ),
+DEFINE_POLICY( REST_NOTOOLBARSONTASKBAR,         "Explorer",       "NoToolbarsOnTaskbar"              ),
+#endif
+DEFINE_POLICY( 0x4000006C,                       "Explorer",       "NoDriveTypeAutoRunCommandAndVerb" ),
+#if (NTDDI_VERSION >= NTDDI_WINXP)
+DEFINE_POLICY( REST_NODISPBACKGROUND,            "System",         "NoDispBackgroundPage"             ),
+DEFINE_POLICY( REST_NODISPSCREENSAVEPG,          "System",         "NoDispScrSavPage"                 ),
+DEFINE_POLICY( REST_NODISPSETTINGSPG,            "System",         "NoDispSettingsPage"               ),
+DEFINE_POLICY( REST_NODISPSCREENSAVEPREVIEW,     "System",         "NoScreenSavePreview"              ),
+DEFINE_POLICY( REST_NODISPLAYCPL,                "System",         "NoDispCPL"                        ),
+DEFINE_POLICY( REST_HIDERUNASVERB,               "Explorer",       "HideRunAsVerb"                    ),
+DEFINE_POLICY( REST_NOTHUMBNAILCACHE,            "Explorer",       "NoThumbnailCache"                 ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WINXPSP1) || defined(IE_BACKCOMPAT_VERSION)
+DEFINE_POLICY( REST_NOSTRCMPLOGICAL,             "Explorer",       "NoStrCmpLogical"                  ),
+DEFINE_POLICY( 0x40000095,                       "Explorer",       "NoInternetOpenWith"               ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN2KSP3)
+DEFINE_POLICY( REST_NOSMCONFIGUREPROGRAMS,       "Explorer",       "NoSMConfigurePrograms"            ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WINXPSP1) || defined(IE_BACKCOMPAT_VERSION)
+DEFINE_POLICY( REST_NOPUBLISHWIZARD,             "Explorer",       "NoPublishingWizard"               ),
+DEFINE_POLICY( REST_NOONLINEPRINTSWIZARD,        "Explorer",       "NoOnlinePrintsWizard"             ),
+DEFINE_POLICY( REST_NOWEBSERVICES,               "Explorer",       "NoWebServices"                    ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN2KSP3)
+DEFINE_POLICY( REST_ALLOWUNHASHEDWEBVIEW,        "Explorer",       "AllowUnhashedWebView"             ),
+#endif
+DEFINE_POLICY( REST_ALLOWLEGACYWEBVIEW,          "Explorer",       "AllowLegacyWebView"               ),
+#if (NTDDI_VERSION >= NTDDI_WIN2KSP3)
+DEFINE_POLICY( REST_REVERTWEBVIEWSECURITY,       "Explorer",       "RevertWebViewSecurity"            ),
+#endif
+DEFINE_POLICY( REST_INHERITCONSOLEHANDLES,       "Explorer",       "InheritConsoleHandles"            ),
+DEFINE_POLICY( REST_NOREMOTERECURSIVEEVENTS,     "Explorer",       "NoRemoteRecursiveEvents"          ),
+#if (NTDDI_VERSION < NTDDI_VISTA)
+DEFINE_POLICY( REST_SORTMAXITEMCOUNT,            "Explorer",       "SortMaxItemCount"                 ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WINXPSP2)
+DEFINE_POLICY( REST_NOREMOTECHANGENOTIFY,        "Explorer",       "NoRemoteChangeNotify"             ),
+#endif
+DEFINE_POLICY( 0x4000009C,                       "Explorer",       "AllowFileCLSIDJunctions"          ),
```

Comment:
Remove this one, because it duplicates the REST_ALLOWFILECLSIDJUNCTIONS from below.

Reply:
Deleted.